x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Meta } from '../../../../../common/types';
+
+export interface ApiLog {
+  timestamp: string; // Date ISO string
+  status: number;
+  http_method: string;
+  full_request_path: string;
+  user_agent: string;
+  request_body: string; // JSON string
+  response_body: string; // JSON string
```

Comment:
FYI we also get back
```
path: null,
```
from our API but I have no idea what it does and we're not using it at all in the standalone UI, so I ignored it. I could have noted it in a comment, but I figured maybe github history (this comment) is sufficient

Reply:
https://github.com/elastic/kibana/pull/95732/commits/f87d87153bc0f68642a074d3a7c59bcd231b01f4